lib/expectations_runner.rb

```diff
@@ -19,9 +25,18 @@ class Mumukit::Inspection::PlainInspection
 
   def expectations
     {
-      'HasWhile' => /AST\(while/
+      'HasWhile' => use(/AST\(while/),
+      'HasBinding' => lambda { |binding| subject_for(binding).ast_regexp }
     }
   end
+
+  def subject_for(binding)
+    if binding == 'program'
```

Comment:
This is a bit ugly, but `Subject.from` returns `Program` only when the binding is empty. Thinking it again, I could change it to consider this case (right now it returns `Function` for input `string` and that's not right).